File: conditionals/time_of_day.py
# Adelle Ponczoch, Time of Day in Python

import time
current = time.time()
local_time = time.localtime(current)
hour = local_time.tm_hour

if hour >= 12 and hour < 19:
    print("Good afternoon!")
elif hour >=19:
    print("Good evening!")
else:
    print("Good morning!")

File: conditionals/time_of_day.c
//Adelle Ponczoch, Time of Day in C
#include <stdio.h>
#include <time.h>

int main(void){
    time_t seconds;
    seconds = time(NULL);
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;

    if(hour >= 12 && hour < 19){
        printf("Good afternoon!");
    }else if(hour >= 19){
        printf("Good evening!");
    }else{
        printf("Good morning!");
    }
    return 0;
}